config/routes.rb

```diff
@@ -1,14 +1,21 @@
 Rails.application.routes.draw do
-  devise_for :users
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   scope '(:locale)', locale: /#{I18n.available_locales.join("|")}/, defaults: { locale: I18n.default_locale } do
+    devise_for :users
+
     root 'home#index', as: 'home'
 
     get "/404", to: "errors#not_found"
     get "/422", to: "errors#unacceptable"
     get "/500", to: "errors#server_error"
-    resources :users
-    resources :osbbs
-    resources :billing_contracts
+    resources :osbbs, only: %i[index show]
+    namespace :account do
+      resources :users, except: %i[index destroy]
+      namespace :admin do
+        resources :osbbs, except: %i[index show]
+        get '/user_cabinets', to: 'user_cabinets#index', as: 'user_cabinet'
```

Comment:
why you need this mapping?

Reply:
as: 'user_cabinet'? I did it without reason) Delete it?